src/components/molecules/TableComponent/TableComponent.tsx

```diff
@@ -82,6 +82,32 @@ const TableComponent: React.FunctionComponent<TableComponentPropsType> = ({
 
   const itemClasses = classNames("TableComponent__item", itemStyles);
 
+  const userPictures = useMemo(
```

Comment:
In line with other uses in the project, `rendered` prefix provides visual cue for the intended use.

```suggestion
  const renderedUserPictures = useMemo(
```

Reply:
done https://github.com/sparkletown/sparkle/pull/2760/commits/9cc6e165b25244d4599206263d7e1e47ab5f583f